File: src/main/Java/boj_step5/boj2908.java
import java.io.*;
import java.util.StringTokenizer;

public class boj2908 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int a = Integer.parseInt(new StringBuilder(tokenizer.nextToken()).reverse().toString());
        int b = Integer.parseInt(new StringBuilder(tokenizer.nextToken()).reverse().toString());
        writer.append(String.valueOf(Math.max(a, b)));
        writer.close();
    }
}

File: src/main/kotlin/others/boj2908.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import kotlin.math.max

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    var (a, b) = r.readLine()!!.split(" ").map { it }
    a = a.reversed()
    b = b.reversed()
    w.write(max(a.toInt(), b.toInt()).toString())
    w.close()
}